### PR TITLE
update deps links for notation-go

### PR DESCRIPTION
**Pull request by dtzar**

### CODE REVIEW

**Comment by shizhMSFT** [comment]
Most codes in `notation-go` will be moved to `notation-core-go`. If we are not in a hurry, can we wait for the code move first?

**Comment by dtzar** [comment]
@shizhMSFT - IMO We should fix what is broken now and then again once things are moved. Main branch should always be in a buildable state to the corresponding upstream deps.

**Review by shizhMSFT — APPROVED** [review]
LGTM

File: go.mod [hunk 1]
```diff
@@ -8,7 +8,7 @@ require (
 	github.com/Azure/go-autorest/autorest/adal v0.9.17
 	github.com/AzureCR/go-jwt-azure v0.0.0-20210928134644-4f31f5fd7906
 	github.com/kelseyhightower/envconfig v1.4.0
-	github.com/notaryproject/notation-go-lib v1.0.0-alpha-1
+	github.com/notaryproject/notation-go v0.7.0-alpha.1.0.20220422005131-96df588f2f6d
```
Comment (shizhMSFT) [inline go.mod:11]: We can now update `notation-go` to `v0.8.0-alpha.1`.

**Pull request merged**